src/assets/styles/_animation.scss

```diff
@@ -71,3 +71,16 @@
 .calcite-animate__in-scale {
   animation-name: in-scale;
 }
+
+@media (prefers-reduced-motion: reduce) {
+  .calcite-animate {
+    opacity: 1;
+    animation: none;
```

Comment:
Is this needed if the duration factor is modified below? If truly needed, can you move this into the block below?

Reply:
This got me thinking and I don't think any component is leveraging animation global classes including `calcite-animate` class. We may not even require this.

Reply:
Consumers do - really we could as well. I'd imagine this comes from `calcite-styles` at some point instead of from CC. It does seem like we should have "from left" and "from right" animations as well...

Reply:
Makes sense. I updated the media query as per your suggestion.